### ISSUE TITLE
Unable to install on Android 12

### ISSUE THREAD

**Issue by skittles9823** [chatsen/chatsen#165]
```adb install .\Chatsen.apk
Performing Streamed Install
adb: failed to install .\Chatsen.apk: Failure [INSTALL_PARSE_FAILED_MANIFEST_MALFORMED: Failed parse during installPackageLI: /data/app/vmdl1022077841.tmp/base.apk (at Binary XML file line #66): com.dexterous.flutterlocalnotifications.ScheduledNotificationBootReceiver: Targeting S+ (version 31 and above) requires that an explicit value for android:exported be defined when intent filters are present]```

**Comment by basedbidoof** [comment]
Also can't update/install on pixel 5 up-to-date android 12. Went to update, wouldn't update, tried uninstalling to reinstall, and now I can't reinstall it.

**Comment by williambulin** [comment]
Seems to be an issue caused by the dependency used for notifications, i'd have to look into it
I couldn't reproduce on my OnePlus 5T running Android 12 (PixelExperience) however

**Comment by skittles9823** [comment]
In running the current Android 12L beta on my Pixel 3a XL, the issue might be limited to pixel devices, but it is blocking me from installing the app from the playstore which is odd.

**Comment by GAMIS65** [comment]
I'm also unable to install the app on Android 12 (Samsung Galaxy S21).

**Comment by williambulin** [comment]
Fixed in 1.1.73+73.
